### PR TITLE
fix: support for java primitives in path variables

### PR DESCRIPTION
**Pull request by aludwiko**
References https://github.com/lightbend/kalix/issues/8053

### CODE REVIEW

File: sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package com.example.wiring.actions.echo;
+
+import kalix.javasdk.action.Action;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class ActionWithPrimitives extends Action {
+
+  @GetMapping("/action/{doubleValue}/{floatValue}/{intValue}/{longValue}")
```
Comment (octonato) [inline sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java:26]: can we have a boolean in the path just for completeness?

**Review by octonato — APPROVED** [review]
LGTM, with a small request

File: sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java (hunk 1 above)
Comment (aludwiko) [reply sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java:26]: there is a `booleanValue` as a query param, should I add it also for path param?

Comment (octonato) [reply sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java:26]: No, not needed. I see that `pathParamExtractors` also calls the new `toExtractor`. So we are covered.

I will merge it.

**Pull request merged**